lib/vacuum/matcher.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Vacuum
+  # A custom VCR matcher
```

Comment:
This is a first time I see such a cool custom matcher for VCR! But since this class is not used in Vacuum library itself, I'd move it to `test/` folder.

Reply:
I put it there so it is available when testing apps that use Vacuum. You just [require](https://github.com/hakanensari/vacuum/blob/improve-v5/test/integration_helper.rb#L8) and [set up VCR](https://github.com/hakanensari/vacuum/blob/improve-v5/test/integration_helper.rb#L14). We can mention this in the README.

Reply:
I brought HTTPI back in after exposing the adapter to make it swappable and realising I was reinventing the wheel.